### ISSUE TITLE
Demo: Update Names of some Background Layers & Drop OSM B&W

### ISSUE THREAD

**Issue by brentfraser** [geomoose/gm3#503]
The tile requests now generate CORS errors.

**Comment by brentfraser** [comment]
And rename the "ArcGIS Rest Example" to something like "National Geographic"

**Comment by brentfraser** [comment]
And remove "Reprojected" from "Reprojected Regional Aerial"

**Comment by brentfraser** [comment]
Maybe a better way to explain the significance of the various layers would be using the Metadata tag?

**Comment by tchaddad** [comment]
We should probably also drop the Mapnik OSM as well, as they are reporting tile server overloading issues. 

I can offer up some catalog settings for the free CARTO layers that show OpenStreetMap data in both color and Balck & White as substitutes

**Comment by tchaddad** [comment]
CARTO greyscale

```
<map-source name="cartogrey" type="xyz">
		<layer name="light" status="off">
			<attribution><![CDATA[
				<a href="https://carto.com/" target="_blank">&copy; Carto</a>,&nbsp;<a href="http://www.openstreetmap.org/copyright" target="_blank">&copy; OpenStreetMap contributors</a>.
				]]></attribution>
		</layer>
		<url>https://a.basemaps.cartocdn.com/rastertiles/light_all/{z}/{x}/{y}.png</url>
		<url>https://b.basemaps.cartocdn.com/rastertiles/light_all/{z}/{x}/{y}.png</url>
		<url>https://c.basemaps.cartocdn.com/rastertiles/light_all/{z}/{x}/{y}.png</url>	
		<param name="cross-origin" value="anonymous"/>
	</map-source>
```

CARTO color

	`
	<map-source name="cartocolor" type="xyz">
		<layer name="voyager" status="off">
			<attribution><![CDATA[
				<a href="https://carto.com/" target="_blank">&copy; Carto</a>,&nbsp;<a href="http://www.openstreetmap.org/copyright" target="_blank">&copy; OpenStreetMap contributors</a>.
				]]></attribution>
		</layer>
		<url>https://a.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}.png</url>
		<url>https://b.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}.png</url>
		<url>https://c.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}.png</url>	
		<param name="cross-origin" value="anonymous"/>
	</map-source>	`

**Comment by elil** [comment]
@brentfraser Metadata may be a better way to explain these in production but in the case of the software demo where none of the content has any meaning other than to show an example, I think that contrived explanation in the name is appropriate.  To me, "Reprojected" has more meaning than "Regional Aerial" (regional aerial is just a hint of where and what to look for, the point is an example of reprojecting).  At least that is my thought.

**Comment by brentfraser** [comment]
@elil, good point.   For me the demo site serves a few purposes:
  - showcase of features
  - testing workflows and features
  - a starting point for making my own site
  - system doc

My thought looking at "Reprojected" was:  Where and when were they reprojected?  Last year by the owner?   Just now by the image server (where ever that is)?  By GeoMoose in JavaScript?  Anyway, it left me wanting more information (layer tips? metadata? more doc on the demo?)

**Comment by elil** [comment]
More info would improve this, I just think that the example component should not be lost.

**Comment by brentfraser** [comment]
@elil sounds like we agree.  I'm not really against the "Reprojected" word, I just think we should add more information to the demo.  Links to doc?  Popups with details? XML snippets?  My mind runs  wild with endless possibilities!

**Comment by brentfraser** [comment]
fixed (the important part anyway) in #506
